bciers/apps/reporting/src/app/components/operations/OperationReview.tsx

```diff
@@ -173,6 +149,16 @@ export default function OperationReview({
   const onChangeHandler = (data: { formData: any }) => {
     const updatedFormData = data.formData;
 
+    if (
+      formDataState.operation_report_type !== undefined &&
```

Comment:
note: this check is a bit of a hack until the useEffect is refactored in #436

Reply:
https://github.com/bcgov/cas-reporting/issues/436. Link above goes to an old cas-registration PR :P